apps/condo/domains/meter/schema/fields/index.js

```diff
@@ -0,0 +1,82 @@
+const { Integer, DateTimeUtc, Checkbox, Relationship } = require('@keystonejs/fields')
+
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+const numberOfTariffs = {
+    type: Integer,
+    isRequired: true,
+}
+
+const installationDate = {
+    schemaDoc: 'Date when the meter was installed in the property',
```

Comment:
Because this field is shared across Meter and PropertyMeter, AFAIK, we should not mean only property
```suggestion
    schemaDoc: 'Date when the meter was installed',
```